rtgui/options.cc

```diff
@@ -2229,10 +2229,13 @@ int Options::saveToFile (Glib::ustring fname)
     FILE *f = g_fopen (fname.c_str (), "wt");
 
     if (f == nullptr) {
-        if (options.rtSettings.verbose) {
-            printf ("Options::saveToFile / Error: unable to open file \"%s\" with write access!\n", fname.c_str());
-        }
-
+        std::cout << "Warning! Unable to save your preferences to: " << fname << std::endl;
+#ifndef RAWTHERAPEE_CLI
+        Glib::ustring msg_ = Glib::ustring::compose(M("MAIN_MSG_WRITEFAILED"), fname.c_str());
```

Comment:
More specifically, we should change `Options::save` to return a `bool` status flag, and then check this in the 3 places where it is called (`rtgui/exportpanel.cc`, `rtgui/preferences.cc` and `rtgui/rtwindow.cc`). Then it's easy to retrieve a suitable parent for the dialog